### DISCUSSION TITLE
account is marked as work or school

### DISCUSSION THREAD

**Discussion by daggs1** [community/community#113886] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Greetings,

I have a personal account here, when I try to use it to connect via microsoft to another site I get an error states that my github account is either work or school account when it is neither.
how can I change the account type?

**Comment by benogoulart** [comment]
Hi,

GitHub personal accounts and managed (Enterprise Managed Users) accounts are different account types, and a standard personal account can't be converted into a work or school account, or vice versa.

If you're seeing a message that your GitHub account is a work or school account when signing in to another website using **Sign in with GitHub**, it's likely related to how that third-party application is interpreting your account rather than a setting you can change in GitHub.

Could you provide a bit more information?

* Which website or application are you trying to sign in to?
* What's the exact error message you're seeing?

That will help determine whether the issue is with the third-party application's authentication flow or something specific to your GitHub account.

**Comment by Mehrab-Javadzadeh** [comment]
From what I understand, GitHub doesn't have an option to switch between personal and work or school accounts. The message you're seeing is probably coming from the app you're trying to log in to, not from GitHub. It's worth noting that some applications may have their own settings for different types of accounts, so it's possible that the issue is with the app itself rather than your GitHub account. If you're still having trouble, you might want to check the app's settings or contact their support team to see if they can help you resolve the issue.
A few things you could check:
When you're logging in, be sure to choose the **"Continue with GitHub"** option instead of **"Sign in with Microsoft"** if you see both options available.
- Try the login in a private/incognito browser window to rule out a cached Microsoft session.
If your app works with both personal and work accounts, it might be getting your GitHub login mixed up.

**Comment by daggs1** [comment]
so, after long time, I've tried to login, the site is ms designer, when I tried to log it I got a message that the account is blocked due to activities against ms service agreement.
however, in contrast to what I got when this issue was opened, I got a possibility to prove I'm human and the account was unblocked.
I hope this is fixed now
